include/BoardData.hxx

```diff
@@ -47,10 +47,20 @@ struct DynaBufferBus : BidirMutexes {
 struct UartBus : DynaBufferBus {};
 struct I2cBus : FixBufferBus<32u> {};
 
+typedef struct  {
+    uint8_t nbr:6;
+    uint8_t isActive:1;
+} ServoPin;
+
+typedef struct {
+    ServoPin Pin;
+    volatile unsigned int angle;
```

Comment:
There's a reason `volatile` is proposed to be deprecated for C++2b; we should never need to use it in this project